addons/point_of_sale/static/tests/tours/product_screen_tour.js

```diff
@@ -413,3 +413,30 @@ registry.category("web_tour.tours").add("AutofillCashCount", {
             ProductScreen.cashDifferenceIs(0),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("ProductSearchTour", {
+    checkDelay: 50,
+    steps: () =>
+        [
+            Chrome.startPoS(),
+            Dialog.confirm("Open Register"),
+            ProductScreen.searchProduct("Test Product"),
+            ProductScreen.productIsDisplayed("Test Product 1"),
+            ProductScreen.productIsDisplayed("Test Product 2"),
+            ProductScreen.searchProduct("Test Produt"), // typo
```

Comment:
is this intended? what does the `// typo` comment mean?

Reply:
Oh, I get it. It's for the fuzzy search.

Reply:
Can you put a comment above saying that typo is intended for the fuzzy search?

Reply:
@caburj It's clear now